description/description.shacl.ttl

```diff
@@ -567,6 +647,32 @@
 <#PersonShape> a sh:NodeShape ; 
```

Comment:
Hier gaan we eens moeten nadenken hoe we dit soort gedeelde definities gaan beheren